staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -354,15 +351,10 @@ func (r *crdHandler) getOrCreateServingInfoFor(crd *apiextensions.CustomResource
 	}
 	creator := unstructuredCreator{}
 
-	// convert CRD schema to openapi schema
-	openapiSchema := &openapispec.Schema{}
-	if err := apiservervalidation.ConvertToOpenAPITypes(crd, openapiSchema); err != nil {
-		return nil, err
-	}
-	if err := openapispec.ExpandSchema(openapiSchema, nil, nil); err != nil {
```

Comment:
are we sure this is only about expanding refs?

Reply:
yes,  see [this comment](https://github.com/kubernetes/kubernetes/pull/58438#issuecomment-358860763)